interactive_tester.py

```
@@ -0,0 +1,130 @@
+        print(f"Building model structure and loading weights from: {MODEL_PATH}...")
+
+        try:
+            # STEP 1: Build the empty model from code
+            self.model = build_deep_hybrid_cnn(input_shape=(IMG_SIZE, IMG_SIZE, 1), num_classes=NUM_CLASSES)
+
+            # STEP 2: Load only the weights
+            self.model.load_weights(MODEL_PATH)
+
+            print("✅ Model and weights loaded successfully!")
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)